installer/constants/parser.py

```diff
@@ -28,7 +28,7 @@
 
 # Other variables
 COMMON_USAGE_HELP_MESSAGE = 'Run \"./xmipp -h\" for usage help.'
-DEFAULT_BUILD_DIR = 'build'
+DEFAULT_BUILD_DIR = 'dist'
```

Comment:
I can rename it for make it more understandable, but as far as I know this variable is never used except by modelsDLTK. 
But @oierlauzi please review it because maybe cmake handle it somehow

Reply:
I just modified the DEFAULT_MODELS_DIR variable

Reply:
You are right about the default build dir potentially not being used, however, CMake is in fact [installing into `/build` directory](https://github.com/I2PC/xmipp/blob/agm_getModels/installer/constants/main.py#L57). The variable `DEFAULT_BUILD_DIR` can be safely removed, but I'd assign `INSTALL_PATH` to the `DEFAULT_MODELS_DIR` variable, as it is already defined there:
```py
from .main import INSTALL_PATH

...
DEFAULT_MODELS_DIR = INSTALL_PATH
...
```

Reply:
I like it!